guide/interactions/modals.md

```diff
@@ -96,7 +96,25 @@ client.on('interactionCreate', interaction => {
 	console.log(interaction);
 });
 ```
+## Responding to modal submissions
 
+The `MessageComponentInteraction` class provides the same methods as the `CommandInteraction` class. These methods behave equally:
+- `reply()`
+- `editReply()`
+- `deferReply()`
+- `fetchReply()`
+- `deleteReply()`
+- `followUp()`
+
+```js{1,4-6}
+client.on('interactionCreate', interaction => {
+	if (!interaction.isModalSubmit()) return;
+
+	if (interaction.customId === 'myModal') {
+		await interaction.reply({ content: 'Your submit recieved!' });
```

Comment:
```suggestion
		await interaction.reply({ content: 'Your submission was recieved successfully! });
```
The other sections do not check for customId so the surrounding if statement might not be consistent?

Reply:
I mean as in the other sections in the modal page

Reply:
Hmm I don't know, maybe they need too